schedule.html

```diff
@@ -0,0 +1,323 @@
+      </li>
+      <li>
+        <a href="#program">Program Committee</a>
+      </li>
+      <li>
+        <a href="papers.html">Position Statements</a>
+      </li>
+      <li>
+        <a class="active-tab">Workshop Schedule</a>
+      </li>
+    </ul>
+    <aside class="box" id="host">
+      <h2 class="footnote">
+        Host
+      </h2>
+      <p>
+        W3C gratefully acknowledges Samsung for hosting this workshop.
+      </p>
+      <p>
+        <a href="http://samsung.com/"><img src="samsung.svg" alt="Samsung"></a>
+      </p>
+    </aside>
+    <aside class="box" id="sponsor">
+      <h2 class="footnote">
+        Sponsors
+      </h2>
+      <article class="sponsors">
+        <h3 id="platinum-sidebar">
+          Platinum Sponsors
+        </h3>
+        <p>
+          <a href="https://www.google.com/"><img src="google.svg" alt=
+          "Google"></a><br>
+          <a href="https://www.mozilla.org/"><img src="mozilla.svg" alt=
+          "Mozilla"></a>
+        </p>
+      </article>
+      <p>
+        <a href="./#sponsorship">Become a sponsor!</a>
+      </p>
+      <p>
+        <a href="sponsorship.html">Sponsorship package info.</a>
+      </p>
+    </aside>
+    <main>
+      <h2 id="schedule">
+        Tentative Schedule
+      </h2>
+      <p>
+        The workshop will focus around several topics identified by the
+        <a href="https://www.w3.org/2002/09/wbs/1/vr-workshop/results">expressions
+        of interest</a>. Each topic will be introduced by one or more related
+        lightning talks, and will be explored more in-depth by discussion
+        breakouts, concluded with joint summaries of the breakouts. The goal of
+        each discussion is not to resolve the technical issues of the topic,
+        but to determine its relevance and priority to standardization.
+      </p>
+      <p>
+        <b>This schedule may change based on discussions with the program
+        committee.</b>
+      </p>
+      <h4 id="schedule-day-1">
+        Day 1: Wednesday, October 19, 2016
+      </h4>
+      <table class="schedule">
+        <tbody>
+          <tr>
+            <th>
+              08:00–08:30 AM
+            </th>
+            <td>
+              Registration
+            </td>
+          </tr>
+          <tr>
+            <th>
+              08:30–08:45 AM
+            </th>
+            <td>
+              Opening remarks by <a href=
+              "https://twitter.com/dontcallmeDOM">Dominique Hazael-Massieux</a>
+            </td>
+          </tr>
+          <tr>
+            <th>
+              08:45–09:00 AM
+            </th>
+            <td>
+              Keynote: (to be announced)
+            </td>
+          </tr>
+          <tr>
+            <th>
+              09:00–09:45 AM
+            </th>
+            <td>
+              WebVR intro: implementation status, obstacles, future plans
+            </td>
+          </tr>
+          <tr class="break">
+            <th>
+              09:45–10:00 AM
```

Comment:
in general (i.e. not only), 15 minutes seems very short for the breaks, especially given that there will be 100+ people there; I believe a lot of the outcomes of the workshop will hinge on less formal discussions too, so I would prefer if we organized ourselves around 30 minutes breaks.